packages/flutter/lib/src/material/flat_button.dart

```diff
@@ -203,8 +273,21 @@ class FlatButton extends MaterialButton {
     double height,
   }) = _FlatButtonWithIcon;
 
-  @override
-  Widget build(BuildContext context) {
+  final _FlatButtonType _buttonType;
+
+  Widget _buildCupertinoButton(BuildContext context) {
+    return CupertinoButton(
+      key: key, 
+      onPressed: onPressed, 
+      color: color, 
+      disabledColor: disabledColor ?? CupertinoColors.quaternarySystemFill, 
```

Comment:
I think you don't have to do the ?? here. If it's null, it'll default to CupertinoColors.quaternarySystemFill in cupertino button.